.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,38 @@
+## Pull Request
+
+### Description
+<!-- Provide a brief description of the changes introduced by this pull request. -->
+
+### Related Issues
+<!-- Mention any related issues that this pull request addresses or resolves. Use the syntax: "Resolves #123" or "Fixes #456". -->
+
+### Checklist
+<!-- Ensure all the following items are completed before submitting the pull request. -->
+- [ ] Code follows the project's coding standards.
+- [ ] Documentation is updated (if applicable).
+- [ ] All checks and tests pass locally.
+- [ ] The commit messages follow the project's [commit message conventions](.
```

Comment:
Missing link, I think you can use: `[commit message conventions](./CONTRIBUTING.md)`

Reply:
This has been resolved with #a2dc51617a120b1f6ce47df99568c3e0e9013aae